src/components/box/box.css

```diff
@@ -1,5 +1,2 @@
 .box {
-    display: flex;
-    position: relative;
-    background-color: #E9EEF2;
-}
+}
```

Comment:
Without `flex`, the `Box` component does not really do anything for us, so perhaps it should be removed and replaced with the native elements?